pkg/graveler/staging/manager.go

```diff
@@ -100,3 +113,38 @@ func (m *Manager) DropByPrefix(ctx context.Context, st graveler.StagingToken, pr
 
 	return nil
 }
+
+func (m *Manager) cleanupLoop(ctx context.Context) {
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		case <-m.wakeup:
```

Comment:
When is it false?

Reply:
You must have a value for a channel and I hate `struct{}` :)